doc/recipe/preprocessor.rst

```diff
@@ -1475,6 +1476,24 @@ Parameters:
     ``'nearest'``. There is no default.
 
 
+``extract_location``
+--------------------
+
+Extract a single point using a location name, with interpolation
+(either linear or nearest). This preprocessor extracts a single
+location point from a cube, according to the given interpolation
+scheme ``scheme``. The function retrieves the coordinates of the
+location and then calls the :func:`esmvalcore.preprocessor.extract_point`
+preprocessor. It can be used to locate cities and villages,
+but also mountains or other geographical locations.
```

Comment:
could you maybe give me the locations you tested with so I can list them as examples please? No lists asa far as I can see, prob best to keep it this way, otherwise it'd be called `extract_locations`?

Reply:
I tested `'ammersee'`, `'münchen'`, `'frankfurt'`, `'frankfurt, oder'`, `'usa'`, `'new york, usa'`, `'watzmann'`, `'mount everest'`. Just pick something you like :smile:

Reply:
Nice! US is a bit bigger than a point, I wonder what actual point it used, Daytona :racing_car: :checkered_flag:

Reply:
OK I added examples :wink: